buildkitd/docker-auto-install.sh

```diff
@@ -110,10 +118,33 @@ install_dockerd_amazon() {
     esac
 }
 
+install_jq() {
+    case "$distro" in
+        alpine)
+            apk add --update --no-cache jq
+            ;;
+
+        amzn)
+            yum install jq
+            ;;
+
+        *)
+            export DEBIAN_FRONTEND=noninteractive
+            apt-get update
+            apt-get install -y jq
+            ;;
+    esac
+}
+
 if [ "$(id -u)" != 0 ]; then
     echo "Warning: Docker-in-Earthly needs to be run as root user"
 fi
 
+if ! detect_jq; then
+    echo "jq is missing. Attempting to install automatically."
+    install_jq
+fi
+
```

Comment:
I think we need to update this script too then: https://github.com/earthly/lib/blob/main/install-dind.sh.